test-apps/display-test-app/package.json

```diff
@@ -83,12 +83,12 @@
     "eslint": "^6.8.0",
     "express": "^4.16.3",
     "fs-extra": "^8.1.0",
-    "internal-tools": "0.0.0",
+    "internal-tools": "workspace:*",
     "npm-run-all": "^4.1.5",
     "null-loader": "^0.1.1",
     "rimraf": "^3.0.2",
     "react": "^16.8.0",
-    "sass": "^1.29.0",
+    "sass": "~1.32.0",
```

Comment:
This change will not be need if  https://github.com/imodeljs/imodeljs/pull/1527 is merged first.

Reply:
thanks @bsteinbk, just reverted the change to use `~`.